tests/ieee802154_hal/Makefile

```diff
@@ -1,5 +1,6 @@
 include ../Makefile.tests_common
-
+TERMPROG := socat
```

Comment:
Do we need to enforce socat here?

Reply:
Yes, because when running the test over Release-Specs the output, without socat, will be:
```
['> ', '[PEXPECT_PROMPT+']
Exception 
```
I guess it is something related to RIOT Ctrl

Reply:
I will try to reproduce the issue here